### PR TITLE
[MIG] hr_holidays_public: Migration to 13.0

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

**Comment by damdam-s** [comment]
cannot see runbot logs because log page leads to 404

**Comment by damdam-s** [comment]
@pedrobaeza seems I cannot force rebuild by myself on runbot

**Comment by pedrobaeza** [comment]
Please add #694

**Comment by pedrobaeza** [comment]
> seems I cannot force rebuild by myself on runbot

Weird, that option is opened for everyone. Maybe the problem is that the button doesn't appear on the detail form, and you have to go to the list and activate it from the dropdown.

About the warnings, now we have them available:

```
2019-10-04 10:08:28,670 149 [1;33m[1;49mWARNING[0m openerp_template odoo.addons.phone_validation.tools.phone_validation: The `phonenumbers` Python module is not installed, contact numbers will not be verified. Please install the `phonenumbers` Python module. 
2019-10-04 10:08:39,249 200 WARNING openerp_test odoo.fields: Field hr.holidays.public.line.variable_date: parameter 'oldname' is not longer supported; use an upgrade script instead.
2019-10-04 10:08:39,665 200 WARNING openerp_test odoo.models: ir.actions.act_window.create() with unknown fields: view_type
```

**Comment by jarroyomorales** [comment]
If this https://github.com/OCA/hr/issues/646 applies also for 13.0 i think it should also be taken into account, but if you prefer to migrate it first and then solve this it's ok for me.

File: hr_holidays_public/__manifest__.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 {
     'name': 'HR Holidays Public',
-    'version': '12.0.1.0.1',
+    'version': '13.0.1.0.0',
     'license': 'AGPL-3',
     'category': 'Human Resources',
     'author': "Michael Telahun Makonnen, "
               "Tecnativa, "
               "Fekete Mihai (Forest and Biomass Services Romania),"
+              "Camptocamp,"
```
Comment (jarroyomorales) [inline hr_holidays_public/__manifest__.py]: Not author, better add yourself as contributor in the README

**Review by jarroyomorales — APPROVED** [review]
Just one comment but overall LGTM, no major changes :+1:

File: hr_holidays_public/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2015 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# Copyright 2019 Camptocamp
```
Comment (emagdalenaC2i) [inline hr_holidays_public/__manifest__.py]:
```suggestion
```

Comment (damdam-s) [reply hr_holidays_public/__manifest__.py]: @emagdalenaC2i any reason? IMHO each work can be credited no?

File: hr_holidays_public/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply hr_holidays_public/__manifest__.py]: Yes, this migration doesn't grant your company as co-author

**Review by pedrobaeza — DISMISSED** [review]
Remove co-authorship

File: hr_holidays_public/__manifest__.py (hunk 2 above)
Comment (emagdalenaC2i) [reply hr_holidays_public/__manifest__.py]: Yeah, of course. But as contributor, IMHO not to be included in the copyright for a simple migration

**Comment by emagdalenaC2i** [comment]
@damdam-s I think you have to rebase and run pre-commit run -a otherwise, ocabot merge will fail.
See https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97

**Comment by emagdalenaC2i** [comment]
@damdam-s Please, could you fix travis pre-commit

File: hr_holidays_public/models/hr_leave_type.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import fields, models
+
+
+class HrLeaveType(models.Model):
+    _inherit = "hr.leave.type"
+
+    exclude_public_holidays = fields.Boolean(
+        string="Exclude Public Holidays",
+        default=True,
+        help=("If enabled, public holidays are skipped in leave days" " calculation."),
```
Comment (emagdalenaC2i) [inline hr_holidays_public/models/hr_leave_type.py]:
```suggestion
        help=("If enabled, public holidays are skipped in leave days calculation."),
```

**Review by emagdalenaC2i — APPROVED** [review]
LGTM just a minor change and please squash your commits

**Comment by damdam-s** [comment]
@emagdalenaC2i suggestion + squash done
@pedrobaeza changes requested done. Can you update your review?
Thanks

**Comment by emagdalenaC2i** [comment]
@pedrobaeza or @jbeficent please, could you review this PR?

**Comment by pedrobaeza** [comment]
@damdam-s can you rebase in your local and run pre-commit that is failing?

**Comment by pedrobaeza** [comment]
Can you please move this to OCA/hr-holidays following #732?

**Comment by ernestotejeda** [comment]
@damdam-s, if you want I can move this PR to https://github.com/OCA/hr-holidays to speed up the migration of this module a bit and also unblock https://github.com/OCA/hr-attendance/pull/1. What do you think?

**Comment by pedrobaeza** [comment]
He's not working anymore at camptocamp, so you can go on with the superseding.

**Comment by damdam-s** [comment]
@ernestotejeda sorry for not answering this... 

I'm quite busy on other topics these days. Of course, you can superseed 😄 

@pedrobaeza You're right but that doesn't mean I won't be active anymore on OCA 😃 . For now, it's just a matter of priorities ...

**Comment by pedrobaeza** [comment]
Oh, good to know that you are still in Odoo's and OCA's world! I thought we lost you!

**Comment by pedrobaeza** [comment]
Superseded then by OCA/hr-holidays#1

**Pull request closed without merging**